chia/server/rate_limit_numbers.py

```diff
@@ -111,6 +111,7 @@ def compose_rate_limits(old_rate_limits: Dict[str, Any], new_rate_limits: Dict[s
             ProtocolMessageTypes.request_peers: RLSettings(10, 100),
             ProtocolMessageTypes.respond_peers: RLSettings(10, 1 * 1024 * 1024),
             ProtocolMessageTypes.request_puzzle_solution: RLSettings(1000, 100),
+            ProtocolMessageTypes.request_puzzle_solution_with_coin_info: RLSettings(1000, 100),
```

Comment:
I think we should allow more than 1000 per minute here. I would think closer to 3000 (that's 50 per second).

Reply:
@mariano54 thoughts?

Reply:
Agreed,  this might become a bottleneck in syncing cats,  if you can only do 50/second.
Its a complicated issue because we have to balance DOS attacks as well as wallet performance. I'm not sure